packages/patternfly-4/react-core/src/components/Card/examples/Card.md

```diff
@@ -75,7 +75,9 @@ class KebabDropdown extends React.Component {
     return (
       <Card>
         <CardHead>
-          <img src={pfLogo} alt="PatternFly logo" style={{ height: '50px' }}/>
+          <CardHeadMain>
+            <img src={pfLogo} alt="PatternFly logo" style={{ height: '50px' }}/>
+          </CardHeadMain>
```

Comment:
I know `img` already exist here, but shouldn't we use the Brand component for the PatternFly logo? That way, we don't need to specify the image path.

<Brand src={pfLogo} alt="Patternfly Logo" />